debug/test/browser/debug.test.js

```diff
@@ -275,6 +275,21 @@ describe('debug', () => {
 			expect(serializeVNode(<Foo />)).to.equal('<Foo />');
 		});
 
+		it('should serialize vnode without props', () => {
+			const vnode = {
+				type: null,
+				props: null,
+				text: 'hello world',
+				key: null,
+				ref: null
+			};
+				type: 'string',
```

Comment:
I think this is a syntax error

Reply:
You're right. I trusted github to apply the change but shouldn't have. :)